### PR TITLE
Created the readme file,html,CSS

### PR DESCRIPTION
**Pull request by Halim258**
Below are listed allthe features included in this PR:
 

- [ ] Add linters configuration
- [ ]  Add index.html file
- [ ]  Add styles.css file
- [ ]  Add a README file

### CODE REVIEW

File: .github/workflows/linters.yml
Comment (AbelG101) [inline .github/workflows/linters.yml:1]: - [ ] Kindly make sure your pull Request has a summary describing introduced changes in more detail than the current one as this helps viewers get a quick understanding of the changes that were added. 😉
- Here's how it looks: 
  image

File: README.md [hunk 1]
```diff
@@ -1,2 +1,115 @@
...
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
```
Comment (AbelG101) [inline README.md:64]: - [ ] Please make sure you only include the tech stack that you're using for the project like `HTML`, `CSS` & `JS`. As this makes your project looks unprofessional and a bit reckless. 😟

File: README.md [hunk 2]
```diff
@@ -1,2 +1,115 @@
...
+
+- **Feature 1:** [ this application that I'm going to do will allow users to Pay the service online]
+- **Feature 2:** [ features #2 it will also allow them I'm in the users to connect with each others in the chat box]
+- **Feature 3:** [ this will also include different mind games such as chess Rubik's Cube]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://example.com)
```
Comment (AbelG101) [inline README.md:76]: - [ ] Please make sure to remove the "Live demo" section from your readme as it's not required for this project (please take note this only applies to this PR as adding a demo link isn't required but for future projects, this section should exist).

File: README.md [hunk 3]
```diff
@@ -1,2 +1,115 @@
...
+
+### Run tests
+
+To run tests, run the following command:
+
+### Deployment
+
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (AbelG101) [inline README.md:108]: - [ ] Kindly make sure to add clear and detailed steps on how to get started with your project such as the prerequisites, setting up, etc... the default text should be removed 🎯
- [ ] You can also remove the `Deployment` heading as it's not required to deploy the project.

File: README.md
Comment (AbelG101) [inline README.md:1]: - [ ] Kindly take note that the main headings should not be removed from your readme file except for the FAQ which is always optional and the "Live demo" section which is sometimes okay to remove if it's not a project requirement to deploy your web app. But the rest of the headings like the 🔭 Future Features, 🤝 Contributing, ⭐️ Show your support, 🙏 Acknowledgements, ❓ FAQ (OPTIONAL) and 📝 License should not be removed
- [ ] Please also take caution that under the License heading you should place a link to a license file which you can [find here](https://github.com/microverseinc/readme-template/blob/master/MIT.md) (so first add the license to your root directory) and make sure the link points to that file using `./[repalce-wtih-file-name]`

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Halim258 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title✔

## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bantealem — APPROVED** [review]
Hi @Halim258 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Congratulations-Memes-FEATURED

## Highlights
:heavy_check_mark:  Descriptive PR
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark:  Correct Github flow

## Optional suggestions
- [ ] Kindly create the `MIT` license when you create a repo. This will help you to get the link automatically

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**